### PR TITLE
Feature/ch122277/list component meta value

### PR DESCRIPTION
**Pull request by builtbyedgar**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 ## Unreleased
 - Make OAuthLogin component responsive [#28](https://github.com/CartoDB/carto-react-lib/pull/28)
 
+## 1.0.0-beta5 (2020-11-26)
+- Added meta value version to List component [#31](https://github.com/CartoDB/carto-react-lib/pull/31)
```
Comment (neokore) [inline CHANGELOG.md:5]: Move this item to ##Unreleased section, please

**Review by neokore — COMMENTED** [review]
LGTM, move the changelog item and let's merge it

File: CHANGELOG.md (hunk 1 above)
Comment (builtbyedgar) [reply CHANGELOG.md:3]: done

**Review by neokore — APPROVED** [review]
LGTM

**Pull request merged**